src/main/java/Duke.java

```diff
@@ -1,10 +1,189 @@
+import java.util.*;
+
 public class Duke {
+
+    static void level1() {
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
+
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+
+        while (!input.equals("bye")) {
+            System.out.println(input);
+            input = sc.nextLine();
+        }
+
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+
+    static void level2() {
+        System.out.println("Hello! I'm Duke\n" + "What can I do for you?");
+
+        Scanner sc = new Scanner(System.in);
+        String input = sc.nextLine();
+
+        List<String> lst = new ArrayList<>();
+
+        while (!input.equals("bye")) {
+            if (!input.equals("list")) {
+                lst.add(input);
+                System.out.println("added: " + input);
+            } else {
+                for (int i = 1; i <= lst.size(); i++) {
+                    System.out.println(i + ". " + lst.get(i-1));
+                }
+            }
+            input = sc.nextLine();
+        }
+
+        System.out.println("Bye. Hope to see you again soon!");
+
+    }
+
+
+    static void level3() {
+        List<Task> lst = new ArrayList<>();
```

Comment:
Perhaps instead of lst, using tasks could be clearer to know what the list contains.